gflownet/policy/simple_tree.py

```diff
@@ -0,0 +1,163 @@
+from typing import Optional
+
+import torch
+import torch_geometric
+from torch_geometric.data import Batch
+from torch_geometric.nn import global_add_pool
+
+from gflownet.envs.tree import Tree
```

Comment:
Rather minor. Since only the (static) method `Tree.to_pyg()` is used in this class, shall we instead import only this method instead of the whole class? 

And actually I would rename `Tree.to_pyg()` as `Tree.state_to_pyg()` or `Tree.state2pyg()` to make it more explicit. But just my personal preference.

Reply:
Thanks! Right right that was stupid of me...